packages/api/src/repository.js

```diff
@@ -12,28 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const { Firestore } = require('@google-cloud/firestore');
 const moment = require('moment');
+const client = require('./firestore.js');
```

Comment:
if what you are trying to avoid is running the New Firestore() code multiple times, than this is not fixing that. It is my understanding that everytime you run require it will generate a new client object. The design pattern would be generating client once and passing that in in functions.

Reply:
Good catch, I hadn't fully tested that! I just did by adding print statements in firestore.js, and in repository.js and server.js right after each of them required firestore.js, and while print statements from repository and server both appeared on the console, the print statement in firestore only printed once. This seems to show that the code is only run the first time firestore.js is called, and after that it simply exports the calculation it already did.